### PR TITLE
KOGITO-2717: Use random ports for Quarkus and Spring Boot apps in Tests

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA: https://issues.redhat.com/browse/KOGITO-2717
Descrption: Use random ports for Quarkus and Spring Boot tests.

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by Sgitario** [comment]
Depends on https://github.com/kiegroup/kogito-examples/pull/296

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by Sgitario** [comment]
jenkins please retest this

File: process-timer-quarkus/src/test/java/org/acme/travels/it/ProcessTimerIT.java [hunk 1]
```diff
@@ -93,13 +97,13 @@ public void testDeleteBoundaryTimersOnTask() {
...
 
-    private ValidatableResponse getTimerWithStatusCode(String id, int code, String path) {
+    private ValidatableResponse getTimerWithValidStatus(String id, String path) {
         return given()
                 .accept(ContentType.JSON)
                 .when()
                 .get("/" + path + "/{id}", id)
                 .then()
-                .statusCode(code);
+                .statusCode(anyOf(is(HttpStatus.SC_OK), is(HttpStatus.SC_NO_CONTENT)));
```
Comment (Sgitario) [inline process-timer-quarkus/src/test/java/org/acme/travels/it/ProcessTimerIT.java:106]: It seems that Jobs Service is returning either 200 or 204. I've added this as a temporal fix as these tests will be done in Jobs Service apps.

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by cristianonicolai — APPROVED** [review]

**Review by jstastny-cz — APPROVED** [review]

**Pull request merged**